### PR TITLE
[CN-520] Add operator Jet tutorial to the website

### PR DESCRIPTION
**Pull request by cagric0**
# Description of change

Operator Jet tutorial is added

## Type of change

Select the type of change that you're making:

- [ ] Bug fix (Addresses an issue in existing content such as a typo)
- [x] Enhancement (Adds new content)

## Open Questions and Pre-Merge TODOs
- [ ] Use github checklists to create a list. When an item is solved, check the box and explain the answer.

### CODE REVIEW

**Comment by cagric0** [comment]
Could you please not merge this PR before merging [the tutorial content PR](https://github.com/hazelcast-guides/hazelcast-platform-operator-jet/pull/1)?

**Review by angelasimms — APPROVED** [review]
LGTM.

**Pull request merged**
